src/testUtils.ts

```diff
@@ -203,16 +223,23 @@ export async function makeFreshTestBed(
   const coin = await pact.fetchAsset(coinIndex);
 
   const poolType = options.poolType ?? "CONSTANT_PRODUCT";
-  const appId = await deployContract(
-    account,
-    poolType,
-    algo.index,
-    coin.index,
-    {
-      feeBps: options.feeBps,
-      amplifier: options.amplifier,
-    },
-  );
+
+  const appId = version
```

Comment:
No need for ternary operator, just pass null/undefined as a version

Reply:
DONE